router-bridge/src/worker.rs

```diff
@@ -302,6 +303,30 @@ fn op_crypto_get_random_values(state: &mut OpState, out: &mut [u8]) -> Result<()
     Ok(())
 }
 
+// HeapStats stores values from a isolate.get_heap_statistics() call
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct MemoryUsage {
+    //rss: usize,
+    heap_total: usize,
+    heap_used: usize,
+    external: usize,
+}
+
+// from https://github.com/denoland/deno/blob/897159dc6e1b2319cf2f5f09d8d6cecc0d3175fa/runtime/ops/os/mod.rs#L329
+// tested in planner.rs
+#[op(v8)]
+fn op_runtime_memory_usage(scope: &mut v8::HandleScope<'_>) -> MemoryUsage {
+    let mut s = v8::HeapStatistics::default();
+    scope.get_heap_statistics(&mut s);
+    MemoryUsage {
+        //rss: rss(),
```

Comment:
?

Reply:
this field is in Deno's original MemoryUsage structure, but we don't expose it because it requires a few different ways to gather it across different platforms, and we don't actually need it for the router